### DISCUSSION TITLE
Реактивность для multiple select

### DISCUSSION THREAD

**Discussion by MadZee-M** [moonshine-software/moonshine#1201] [Ideas]
Приветствую! Планируется ли сделать реактивность для мультивыбора в выборках (или когда значение является массивов)?

Допустим есть марки и модели машин (два поля).
Выбираем несколько марок (в поле Марка), (в поле Модели) нужно по этим маркам вывести все его модели. Но на данный момент, из-за ограничения, не получается.

**Comment by lee-to** [comment]
Не знал что не работает! Сделаю фикс, в пн будет релиз
